services/web/server/src/simcore_service_webserver/login/_2fa.py

```diff
@@ -28,7 +28,7 @@
 def _get_human_readable_first_name(user_name: str) -> str:
     full_name = UserNameConverter.get_full_name(user_name)
     first_name = full_name.first_name.strip()[:20]  # security strip
-    return first_name.capitalize()
+    return cast(str, first_name.capitalize())
```

Comment:
also wondering why mypy is not happy with this??

Reply:
To be honest, don't really know. Python type hinting see it as a string, but mypy is complaining about: Returning Any from a function declared to return "str"